urllib3/contrib/pyopenssl.py

```diff
@@ -138,7 +138,19 @@ def _dnsname_to_stdlib(name):
     then on Python 3 we also need to convert to unicode via UTF-8 (the stdlib
     uses PyUnicode_FromStringAndSize on it, which decodes via UTF-8).
     """
-    name = idna.encode(name)
+    def idna_encode(name):
```

Comment:
Let's leave it for now: I don't think it's useful anywhere else at this time, and we make very few API guarantees in the PyOpenSSL stuff so I don't think this will explode in any big way. We can always safely hoist it if we want to.